test/issuer-ocsp-responder.json

```diff
@@ -1,18 +1,38 @@
 {
-  "ocspResponder": {
-    "source": "file:///hierarchy/intermediate-ocsp-rsa.b64",
-    "path": "/",
-    "listenAddress": "0.0.0.0:4003",
-    "issuerCerts": [
-      "/hierarchy/intermediate-cert-rsa-a.pem"
-    ],
-    "timeout": "4.9s",
-    "shutdownStopTimeout": "10s",
-    "debugAddr": "localhost:8010"
-  },
-
-  "syslog": {
-   "stdoutlevel": 6,
-   "sysloglevel": 6
- }
+	"ocspResponder": {
```

Comment:
This is the config for the issuer OCSP responder, which uses a file source, so it shouldn't require a redis stanza. Perhaps the validation for the OCSP responder config is wrong, and the redis stanza needs to be optional?